release-note-script/src/main/java/MergeReleaseNotes.java

```diff
@@ -0,0 +1,240 @@
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.EnumMap;
+import java.util.List;
+import java.util.Map;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * MergeReleaseNotes creates the body of the merged release note for ScalarDB. This script takes
+ * release note bodies for ScalarDB, ScalarDB Cluster, ScalarDB GraphQL, and ScalarDB SQL as
+ * markdown files as its input. And then output the merged release note body to standard output as a
+ * markdown.
+ *
+ * <p>This script is assumed to be executed in a GitHub Actions workflow.
+ *
+ * <p>Note that it is needed that java 11 to execute this script since it is executed as a
+ * Single-File Source-code program.
+ */
+@SuppressWarnings("DefaultPackage")
+public class MergeReleaseNotes {
+
+  private static final String DEBUG = System.getenv("DEBUG");
```

Comment:
OK, I'll revise it to have boolean flag as the CLI parameter. Thank you.

Reply:
Sorry for flipping my state. Although it should be changed, I'd like to keep the code as is because it works without any problem for now. In addition, it is used only for creating release notes when we release the new version of Scalar products, not for production.

I will revise it according to your advice in another PR when I get a chance to revise it (e.g. Kaizen-day).